src/sentry/models/emailthread.py

```diff
@@ -0,0 +1,40 @@
+"""
+sentry.models.emailthread
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2015 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    Model, BaseManager, BoundedBigIntegerField, sane_repr,
+)
+
+
+class EmailThread(Model):
+    email = models.EmailField()
+    model = models.CharField(max_length=50)
```

Comment:
can we make this an enum, otherwise its going to be full of lots of expensive strings